### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by andresgalante**
@srambach @jgiardino This is the PR with the things I've been working on last week.

Sorry for been so long

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,24 +73,21 @@ To watch for changes, re-generate the front-end, and serve it via a BrowserSync
...
 1. Rebase your branch against master
 1. Checkout that branch
 1. Run the build
-1. Publish the script to github-pages in your repo with the command: `./scripts/publish.sh -r <reponame> public` where `<reponame>` is the local name of your repo (eg. bleathem, origin, etc.)
+1. Publish the script to github-pages in your repo with the command: `npm run publish -- -r <reponame> public` where `<reponame>` is the local name of your repo (eg. bleathem, origin, etc.)
 1. The url is included on the Settings page of your repo under GitHub Pages.
 
 ___
 
 Built with :heart: by the [PattrenFly Community](https://www.patternfly.org/community/) and supported by the Red Hat UXD Team.
```
Comment (jgiardino) [inline README.md]: just noticed a minor typo here in "PattrenFly"

Comment (andresgalante) [inline README.md]: thanks, I've fixed it

File: source/scss/_pf-basic-tags.scss [hunk 2]
```diff
@@ -24,6 +24,13 @@ limitations under the License.
...
   h5 &,
   h6 & {
+    // Size of the tag is relative to the font size of the title
     font-size: $font-size-relative-xs;
   }
+
+  // This anchor targets the close icon in dismissable tags
+  a{
+    color: $pf-color-white;
+    margin-left: $pf-spacer-xxxs;
```
Comment (jgiardino) [inline source/scss/_pf-basic-tags.scss]: Not sure if this is a good solution to the alignment issue, but setting `vertical-align: text-bottom;` here (or on the child `<span>`) makes the X vertically centered in the tag block, but also increases the height of the tag block.

**Comment by jgiardino** [comment]
These updates look great! Thanks for doing this!

File: README.md [hunk 3]
```diff
@@ -73,24 +73,21 @@ To watch for changes, re-generate the front-end, and serve it via a BrowserSync
...
 
 Follow this steps to publish your results to [GitHub pages](https://pages.github.com/):
 
-
 1. Pull master
 1. Rebase your branch against master
 1. Checkout that branch
 1. Run the build
-1. Publish the script to github-pages in your repo with the command: `./scripts/publish.sh -r <reponame> public` where `<reponame>` is the local name of your repo (eg. bleathem, origin, etc.)
+1. Publish the script to github-pages in your repo with the command: `npm run publish -- -r <reponame> public` where `<reponame>` is the local name of your repo (eg. bleathem, origin, etc.)
```
Comment (srambach) [inline README.md]: Doesn't a -- need a tag after it?

**Comment by srambach** [comment]
Nice cleanup, @andresgalante.

File: README.md (hunk 3 above)
Comment (andresgalante) [inline README.md]: yes, probably, the script doesn't work. @bleathem whats the issue here?

**Pull request merged**